doc/conda-recipe-generation.md

```diff
@@ -0,0 +1,58 @@
+Conda Recipe Generation
+=======================
+
+## Rationale
+
+See the [documentation on conda-forge and the robotology-superbuild](conda-forge.md) for a general discussion on how the robotology-superbuild and conda can work together. 
+This document describes how to 
```

Comment:
Ran out of energy there? :)

Reply:
Yes. Fixed in https://github.com/robotology/robotology-superbuild/pull/652/commits/f0c615e0a03de7466ff4253b55d23730ec29ad84 .